copado/main/default/classes/UserStory.cls

```diff
@@ -0,0 +1,49 @@
+public class UserStory extends DomainBuilder {
+
+    // CTORS (with 4 parameters and null ?)
+
+    public UserStory() {
+        super(copado__User_Story__c.SObjectType);
+    }
+
+    public UserStory(Project prj) {
+        this();
+        setParent(copado__User_Story__c.copado__Project__c, prj);
+    }
+
+    public UserStory(Release rls) {
+        this((Project) rls.parentByRelationship.get(copado__Release__c.copado__Project__c));
```

Comment:
We need to think of a nice public method to do that instead of just exposing the private map.

Reply:
But very nice solution

Reply:
Sure, we can come up with something better, totally open to suggestions